docs/schemas/itemMeta.json

```diff
@@ -0,0 +1,996 @@
+        },
+        "accept": {
+          "description": "Allowable file mime types",
+          "type": "array",
+          "items": { "type": "string" }
+        },
+        "maxSize": {
+          "description": "Maximum size of the file in bytes",
+          "type": "number",
+          "default": 1000000
+        },
+        "ascpectRatio": {
+          "description": "Required aspect ratio for this image",
+          "type": "string"
+        },
+        "maxWidth": {
+          "description": "Max width for this image",
+          "type": "number"
+        },
+        "maxHeight": {
+          "description": "Max height for this image",
+          "type": "number"
+        },
+        "minWidth": {
+          "description": "Min width for this image",
+          "type": "number"
+        },
+        "minHeight": {
+          "description": "Min height for this image",
+          "type": "number"
+        }
+      }
+    },
+
+    "tas-signature": {
+      "$schema": "http://json-schema.org/draft-04/schema",
+      "id": "http://talentappstore.github.io/tas-tenant-apis/tas-signature",
+      "title": "tas-signature",
+      "description": "A signature question item",
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": ["signature"]
+        }
+      }
+    },
+
+    "tas-document": {
+      "$schema": "http://json-schema.org/draft-04/schema",
+      "title": "tas-document",
+      "description": "A document question item",
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": ["document"]
+        },
+        "accept": {
+          "description": "Allowable file mime types",
+          "type": "array",
+          "items": { "type": "string" }
+        },
+        "maxSize": {
+          "description": "Maximum size of the file in bytes",
+          "type": "number",
+          "default": 1000000
+        },
+        "signatureRequired": {
+          "description": "If the document must be signed",
+          "type": "boolean"
+        }
+      }
+    },
+
+    "tas-audio": {
+      "$schema": "http://json-schema.org/draft-04/schema",
+      "id": "http://talentappstore.github.io/tas-tenant-apis/tas-audio",
+      "title": "tas-audio",
+      "description": "An audio question item",
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": ["audio"]
+        },
+        "maxLength": {
+          "description": "Max length of recording allowed (in minutes)",
+          "type": "number"
+        },
+        "accept": {
+          "description": "Allowable file mime types",
+          "type": "array",
+          "items": { "type": "string" }
+        },
+        "maxSize": {
+          "description": "Maximum size of the file in bytes",
+          "type": "number",
+          "default": 1000000
+        }
+      }
+    },
+
+    "tas-video": {
```

Comment:
Is this for future expansion? We shouldn't add anything we don't know is needed yet, it just creates more likelihood our predictions will be wrong.